browser-ext/content/content.js

```
@@ -0,0 +1,497 @@
+        }
+    }
+
+    handleMessage(request, sender, sendResponse) {
+        const sendErrorResponse = (message) => {
+            sendResponse({ success: false, message });
+        };
+
+        switch (request.action) {
+            case 'clearHighlights':
+                this.removeHighlights();
+                const result = this.autoFillForms(request.cvData);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)